controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
@aslakknutsen For 'move to top' and 'move to bottom', there wont be any change in the request payload. Its just that it would execute a different if control statement -> if Item.ID == 0. The tests that I have written move an Item to the top of list and bottom of the list. So, I have handled this case as well in the tests and I don't think there is a need to write tests explicitly for that.